include/simdutf/implementation.h

```diff
@@ -137,6 +204,17 @@ simdutf_warn_unused size_t convert_valid_utf16_to_utf8(const char16_t * input, s
  */
 simdutf_warn_unused size_t utf8_length_from_utf16(const char16_t * input, size_t length) noexcept;
 
+/**
+ * Compute the number of bytes that this UTF-16LE string would require in UTF-32LE format.
+ *
+ * This function does not validate the input.
+ *
+ * @param input         the UTF-16LE string to convert
+ * @param length        the length of the string in 2-byte words (char16_t)
+ * @return the number of bytes required to encode the UTF-16LE string as UTF-32LE
+ */
+simdutf_warn_unused size_t utf32_length_from_utf16(const char16_t * input, size_t length) noexcept;
```

Comment:
Same question... isn't this just `count_utf16`?

Reply:
Yes, for same reason as above